test/Tests.Infrastructure/ClusterTestBase.cs

```diff
@@ -576,6 +576,8 @@ protected async Task DisposeAndRemoveServer(RavenServer serverToDispose)
                 }
             }
 
+            WaitForClusterTopologyOnAllNodes(clusterNodes);
```

Comment:
Shouldn't this be an async method? We do have WaitForValueAsync, right?

Reply:
done